lib/packages.cf

```diff
@@ -32,13 +32,22 @@ bundle common package_module_knowledge
     windows::
       "platform_default" string => "msiexec";
 @endif
+
+@if minimum_version(3.12.2)
```

Comment:
Probably don't need this guard.

Reply:
This is in a common bundle, no langauge issues related to this one. platform_default is just a variable, fine to set on any version.